FirebaseInAppMessaging/Sources/FIRInAppMessagingPrivate.h

```diff
@@ -18,9 +18,13 @@
 #import "FIRCore+InAppMessaging.h"
 #import "FIRInAppMessaging.h"
 
```

Comment:
Nit: It is recommended to annotate ObjC headers with `NS_ASSUME_NONNULL_BEGIN` / `NS_ASSUME_NONNULL_END` to provide a default nullability annotations for parameters. Just a note, it doesn't have to be changed at this PR.

Reply:
Added these in 72f47b6.